### PR TITLE
add more compatibility checks for Contao versions

### PR DESCRIPTION
**Pull request by fritzmg**
This adds more detailed checks for each Contao version and detail pages, just as with the other features.

### CODE REVIEW

File: check/views/contao4.phtml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+      <p class="explain"><?php echo _('The PHP DOM extension is part of the PHP core and has to be explicitly disabled using the --disable-dom flag. Recompile PHP without the flag.') ?></p>
+    <?php endif; ?>
+  </div>
+  <div class="row">
+    <h3><?php echo _('PHP intl extension') ?></h3>
+    <?php if ($this->hasIntl()): ?>
+      <p class="confirm"><?php echo _('The PHP intl extension is enabled.') ?></p>
+    <?php else: ?>
+      <p class="error"><?php echo _('The PHP intl extension is not enabled.') ?></p>
+      <p class="explain"><?php echo _('The PHP intl extension is bundled the PHP core since PHP 5.3 and has to be explicitly enabled using the --enable-intl flag. Recompile PHP with the flag.') ?></p>
```
Comment (xchs) [inline check/views/contao4.phtml]: […] is bundled **in** the PHP core […]

Comment (fritzmg) [inline check/views/contao4.phtml]: Right :). Or maybe rather […] is bundles _with_ the PHP core […]

Comment (xchs) [inline check/views/contao4.phtml]: Yep!

**Comment by fritzmg** [comment]
-_- tried to merge with master but I messed it up apparently. Not sure what I did wrong.

**Pull request closed without merging**